runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -156,6 +158,13 @@ else if (config.lvl() && message.toLowerCase().startsWith("!lvl") && message.len
 			log.debug("Running level lookup for {}", search);
 			executor.submit(() -> playerSkillLookup(setMessage.getType(), setMessage, search));
 		}
+		else if (config.calc() && message.toLowerCase().startsWith("!calc") && message.length() > 6)
```

Comment:
I don't use chat commands often, but isn't the syntax supposed to be `::` for "self" type of commands, and `!` for commands that the player actually says? (It should be `::` right?)

Reply:
I believe this commend is intended to share with other people, otherwise i see little point in adding it.

Reply:
Yup